### ISSUE TITLE
[Question]: Better handover xarray warning messages (eg, duplicate dimensions)

### ISSUE THREAD

**Issue by etienneschalk** [etienneschalk/scientific-data-viewer#100]
### Question Type

File format support

### Your Question

The xarray data model does not like duplicate dimensions. However, this is supported in NetCDF, and some scientific compuations rely on using duplicate dimensions (eg variance-covariance matrices).

When opening a NetCDF file with duplicate dimensions, xarray prints a warning. However, this warning is not shown in the extension, and it can be confusing (incoherent dimensions shown in the different views)

More generally, any warning that appears when opening a file should be shown in the viewer, so that the user is aware of potential pitfalls with the opened file. Indeed the extension aims to be as transparent as possible, and any warning or error should be passed down from xarray to the extension. 

### Code of Conduct

- [x] I have searched existing issues and this is not a duplicate
- [x] I have read and agree to follow the [Contributing Guidelines](CONTRIBUTING.md)
